### PR TITLE
update ymls

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 

**What change(s) does this PR introduce?** 

**Did you update the CHANGELOG?** 

- [ ] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] Buildkite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/src_shopify.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ sources:
     tables:
       - name: order
         description: Each record represents an order in Shopify.
-        identifier: "{%- if target.type == 'snowflake' -%} ORDER {%- else -%} order {%- endif -%}"
-        quoting:
-          identifier: true
+        identifier: >
+          {% if target.type == 'snowflake' %}{{ var('shopify_order_identifier', '"ORDER"') }}
+          {% else %}{{ var('shopify_order_identifier', 'order') }}{% endif %}
```
Comment (fivetran-jamie) [inline models/src_shopify.yml:12]: is quoting necessary for the other warehouses though?

Comment (fivetran-jamie) [reply models/src_shopify.yml:12]: like should it be 
```suggestion
        identifier: "{%- if target.type == 'snowflake' -%} {{ var('shopify_order_identifier', 'ORDER' ) }} {%- else -%} {{ var('shopify_order_identifier', 'order') }} {%- endif -%}"
        quoting:
          identifier: true
```

**Review by fivetran-jamie — COMMENTED** [review]
looks good but i have a suggestion for the order identifier part!

File: models/src_shopify.yml (hunk 1 above)
Comment (fivetran-catfritz) [reply models/src_shopify.yml]: Made a small change since the identifier was not working with snowflake when quoting was turned on. It's passing now!

**Comment by fivetran-jamie** [comment]
good to merge once databricks passes!

**Review by fivetran-jamie — APPROVED** [review]

**Pull request merged**